perllib/FixMyStreet/Cobrand/UKCouncils.pm

```diff
@@ -381,6 +438,9 @@ sub munge_report_new_bodies {
         my $thamesmead = FixMyStreet::Cobrand::Thamesmead->new({ c => $self->{c} });
         $thamesmead->munge_thamesmead_body($bodies);
     }
+
+    $self->{c}->cobrand->call_hook(munge_overlapping_asset_bodies => $bodies);
```

Comment:
$self->{c}->cobrand is $self, and you don't need call_hook because you know it exists, it's just above :)